eventually/tests/unittests/team/testmodel.py

```diff
@@ -0,0 +1,174 @@
+
+        with mock.patch('django.utils.timezone.now') as mock_time:
+            mock_time.return_value = TEST_TIME
+
+            self.custom_user1 = CustomUser(id=101,
+                                           first_name='john',
+                                           last_name='doe',
+                                           middle_name='eric',
+                                           email='email',
+                                           password='123456')
+            self.custom_user1.save()
+            self.custom_user2 = CustomUser(id=102,
+                                           first_name='andy',
+                                           last_name='smith',
+                                           middle_name='mor',
+                                           email='exmpl@gmail.com',
+                                           password='456789')
+            self.custom_user2.save()
+            self.custom_user3 = CustomUser(id=103,
+                                           first_name='max',
+                                           last_name='lenon',
+                                           middle_name='poul',
+                                           email='test@gmail.com',
+                                           password='147258')
+            self.custom_user3.save()
+
+            members = [self.custom_user1, self.custom_user2, self.custom_user3]
+            owner = self.custom_user1
+            name = 'bananaboys'
+            description = 'krakaziabra'
+            image = 'link1'
+            self.team1 = Team(101, owner=owner, members=members, name=name, description=description, image=image)
+            self.team1.save()
+
+            members = [self.custom_user1]
+            owner = self.custom_user2
+            name = 'bananadogs'
+            description = 'krakaziabra mama'
+            image = 'link2'
+            self.team2 = Team(102, owner=owner, members=members, name=name, description=description, image=image)
+            self.team2.save()
+
+            members = []
+            owner = self.custom_user3
+            name = 'bananacomputer'
+            description = 'krakaziabra mause cat'
+            image = 'link3'
+            self.team3 = Team(103, owner=owner, members=members, name=name, description=description, image=image)
+            self.team3.save()
+
+    def test_team_to_dict(self):
+        """Method that tests `to_dict` method of certain Team instance."""
+
+        test_team_dict1 = {
+            'id': 101,
+            'name': 'bananaboys',
+            'description': 'krakaziabra',
+            'image': 'link1',
+            'created_at': 1508044512,
+            'updated_at': 1508044512,
+            'owner_id': 101,
+            'members_id': [101, 102, 103]
+        }
+
+        test_team_dict2 = {
+            'id': 102,
+            'name': 'bananadogs',
+            'description': 'krakaziabra mama',
```

Comment:
for example, `bananadogs`, `krakaziabra mama` to 'some name1`, `some description1` ..etc.

Reply:
done